tests/test.py

```diff
@@ -109,8 +110,8 @@ def join_children():
 MIN_ABS = -(2 ** 15)
 MAX_ABS = 2 ** 15
 
-
-tmp = "/tmp/input-remapper-test"
+tmp_TemporaryDirectory = tempfile.TemporaryDirectory(prefix="input-remapper-test")
+tmp = tmp_TemporaryDirectory.name
```

Comment:
```suggestion
tmp = tempfile.TemporaryDirectory(prefix="input-remapper-test")
```

Reply:
When it gets garbage collected it cleans up the temporary directory so it needs to stay reachable while the tests are ran.

Reply:
```suggestion
# When it gets garbage collected it cleans up the temporary directory so it needs to
# stay reachable while the tests are ran.
temporary_directory = tempfile.TemporaryDirectory(prefix="input-remapper-test")
tmp = temporary_directory.name
```